README.md

```diff
@@ -43,6 +43,11 @@ curl -L "https://github.com/docker/compose/releases/download/1.25.5/docker-compo
 chmod +x /usr/local/bin/docker-compose
 ```
 
+Create symbolic link to make docker-compose command executable:
```

Comment:
I had to do this both times I built the monitoring server. 
So I added this here as a step.